rhoai-2.22.1.md

```diff
@@ -84,6 +59,33 @@
     - quay.io/opendatahub/llama-stack-k8s-operator@sha256:f8f66ec02e7f59e510cb20eebfe0c1f6d41b62a9276c6e8e703d258f93d49119
     - quay.io/modh/must-gather@sha256:3b63f056d719ecc1ddd980c99272ade594440af03836bc780f8f394a56010d43
 
+# Optional images:
+These images are ... hmm, do we want to document this somehow ... and what can we actually say?
```

Comment:
What about:
Legacy/Optional Images: These images are no longer officially supported but are still provided for convenience. (They may be useful for users who wish to import older resources or maintain compatibility with previous setups.)

Reply:
updated according your suggestion